samples/react-app/src/App.tsx

```diff
@@ -28,22 +27,22 @@ class App extends React.Component<{}, State> {
     this.state = {
       accounts: [],
       subscriptions: [],
-      isLoggedIn: "unknown"
+      isLoggedIn: "unknown",
```

Comment:
```suggestion
      isLoggedIn: "unknown"
```
[NIT] Do you have prettier for your VS Code?

Reply:
Yea, it's prettier that added these. Unfortunately the default for trailing commas changed between v1 and v2 and I must be using v2 here - I can take a look at that and downgrade globally

Reply:
It's at V5 right now..
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
Try upgrading instead.

> By the way, curious why don't we like trailing commas? It makes sorting, adding, and removing properties easy so I always liked it.

I see. Well, my comment was for consistency. I'm not sure of the reasons why azure-sdk-for-js repo picked one way over the other.

Reply:
Sorry I meant the underlying prettier package 😄 https://www.npmjs.com/package/prettier 

And yea, no worries! I definitely want to be consistent - I was just curious if you happen to know